euth/contrib/migrations/0001_delete_a4_models.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.20 on 2024-09-10 15:03
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=[
+                "DROP TABLE IF EXISTS background_task",
```

Comment:
hmm this didn't work for background_task table. Did it work for you @goapunk?

Reply:
will double check, I copied it from mb/a+ though so i assume it should have worked

Reply:
ah, I forgot to remove the package from the installed apps and dependencies